ui-v2/tests/acceptance/components/catalog-filter.feature

```diff
@@ -61,17 +61,29 @@ Feature: components / catalog-filter
     -------------------------------------------------
   Scenario: Filtering [Model] in [Page]
     Given 1 datacenter model with the value "dc1"
-    And 2 node models from yaml
-    ---
-    - ID: node-0
+    And 1 node model from yaml
+    ---
+    ID: node-0
+    Services:
+    - ID: 'service-0-with-id'
+      Port: 65535
+      Service: 'service-0'
+      Tags: ['monitor', 'two', 'three']
+    - ID: 'service-1'
+      Port: 0
+      Service: 'service-1'
+      Tags: ['hard drive', 'monitor', 'three']
+    - ID: 'service-2'
+      Port: 1
+      Service: 'service-2'
+      Tags: ['one', 'two', 'three']
```

Comment:
This is one place where I noticed I was relying on the mocked data form `consul-api-double`, so I added properly dynamically configured mocks here.